google/cloud/bigquery/table.py

```diff
@@ -928,6 +929,27 @@ def mview_refresh_interval(self, value):
             refresh_interval_ms,
         )
 
+    @property
+    def mview_allow_non_incremental_definition(self):
+        """Optional[bool]: Allow non-incremental definition of the materialized view.
+        The default value is :data:`False`.
+        """
```

Comment:
Gonna use the same docstring as found in the [reference docs](https://cloud.google.com/bigquery/docs/reference/rest/v2/tables#materializedviewdefinition), unless there is an important reason to deviate.

```suggestion
        """Optional[bool]: This option declares the intention to construct a
        materialized view that isn't refreshed incrementally.
        The default value is :data:`False`.
        """
```